File: client/Client/Network/Messages/Session/DenyStartGameMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DenyStartGameMessage
    {
        public const string TYPE = "DenyStartGameMessage";
        private string _reason;

        public string reason
        {
            get => _reason;
            set => _reason = value;
        }

        public DenyStartGameMessage()
        {
        }

        public DenyStartGameMessage(string reason)
        {
            this.reason = reason;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/DenyStartGameMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class DenyStartGameMessage {
    public static final String TYPE = "DenyStartGameMessage";

    private String reason;

    public DenyStartGameMessage() {
    }

    public DenyStartGameMessage(String reason) {
        this.reason = reason;
    }

    public String getReason() {
        return this.reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }
}
